tests/Executor/ExecutorSchemaTest.php

```diff
@@ -75,7 +75,7 @@ public function testExecutesUsingASchema() : void
                 'article' => [
                     'type'    => $BlogArticle,
                     'args'    => ['id' => ['type' => Type::id()]],
-                    'resolve' => function ($_, $args) {
+                    'resolve' => function ($root, $args) {
```

Comment:
and why is it root here and not rootValue?

Reply:
Hm yeah let's change that too.

Reply:
Done